### PR TITLE
Replace zip password with built in backup password

### PR DESCRIPTION
**Pull request by ikifar2012**
Remove ZIP Password logic

### CODE REVIEW

File: remote-backup/config.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ options:
   exclude_folders: ""
   exclude_addons: ""
   remote_directory: ""
-  zip_password: ""
+  password: ""
```
Comment (patman15) [inline remote-backup/config.yaml:39]: I suggest calling it "backup_password" to clearly differentiate it from the user password needed for logging in.

File: remote-backup/run.sh [hunk 2]
```diff
@@ -101,10 +101,10 @@ function create-local-backup {
         fi
         bashio::log.info "Creating partial backup"
         bashio::log.debug "Including ${FOLDERS} and ${ADDONS}"
-        slug=$(ha backups new --raw-json --name="${name}" ${ADDONS} ${FOLDERS} | jq --raw-output '.data.slug')
+        slug=$(ha backups new --raw-json --name="${name}" ${ADDONS} ${FOLDERS} --password="${PASSWORD}" | jq --raw-output '.data.slug')
```
Comment (patman15) [inline remote-backup/run.sh:104]: Did you check whether this also works with an empty password?

**Comment by patman15** [comment]
see my comment, I suggest using `backup_password` otherwise, this cleans up a lot. 👍

File: remote-backup/config.yaml (hunk 1 above)
Comment (ikifar2012) [reply remote-backup/config.yaml]: I agree, I have renamed it to `backup_password`

File: remote-backup/run.sh (hunk 2 above)
Comment (ikifar2012) [reply remote-backup/run.sh]: Yes, when an empty string is given it creates a backup without a password

**Pull request merged**